src/ServicePulse.Host/vue/src/composables/serviceLicense.js

```diff
@@ -12,7 +12,7 @@ const upgradeProtectionExpired =
 const trialExpiring =
   '<div class="license-warning"><strong>Non-production development license expiring</strong><div>Your non-production development license will expire soon. To continue using the Particular Service Platform you\'ll need to extend your license.</div><a href="http://particular.net/extend-your-trial?p=servicepulse" class="btn btn-license-warning"><i class="fa fa-external-link-alt"></i> Extend your license</a><a href="#/configuration" class="btn btn-license-warning-light">View license details</a></div>';
 
-export var license = reactive({
+export let license = reactive({
```

Comment:
should be const

Reply:
We reassign `license` later on.
Maybe we should not do that!